content/docs/deployment/csmoperator/drivers/powermax.md

```diff
@@ -180,6 +180,21 @@ Set up the environment as follows:
 - Add initiators from all ESX/ESXis to a host(initiator group)/host group(cascaded initiator group) where the cluster is hosted.
 - Create a secret which contains vCenter privileges. Follow the steps [here](#support-for-auto-rdm-for-vsphere-over-fc) to create the same.
 
+### CSI PowerMax ReverseProxy
+
+CSI PowerMax ReverseProxy is component that will be installed along with the CSI PowerMax driver. For more details on this feature see the related [documentation](https://dell.github.io/csm-docs/docs/csidriver/features/powermax/#csi-powermax-reverse-proxy).
+
+#### Pre-requisites
+Create a TLS secret that holds an SSL certificate and a private key which is required by the reverse proxy server.
```

Comment:
Replace with "Create a TLS secret that holds an SSL certificate and a private key. This is required by the reverse proxy server."